magma/primitives/when.py

```diff
@@ -121,6 +123,15 @@ def _add_default_drivers_to_register_inputs(
         latches.remove(drivee)
 
 
+def _rewire_driven_value_or_values(value_or_values, temp):
+    if isinstance(value_or_values, list):
```

Comment:
temp -> new_value_or_values

Reply:
Since temp is an instance of `type(value)`, we can just call it `new_value` (shouldn't be a list)